ajax/libs/jQuery.Gantt/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jQuery.Gantt",
+  "version": "1.2.2",
+  "description": "jQuery Gantt Chart is a simple chart that implements gantt functionality as a jQuery component.",
+  "filename": "js/jquery.fn.gantt.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mbielanczuk/jQuery.Gantt.git"
+  },
+  "keywords": [
+    "jquery",
+    "gantt",
+    "char"
+  ],
+  "author": {
+    "name": "Marek Bielańczuk",
+    "url": "https://github.com/mbielanczuk"
+  },
+  "license": "MIT",
```

Comment:
@PeterDaveHello About the license, since the author hasn't replied the issue on git repo for a long time, I sent an email two weeks ago, and get no response. Can we use `MIT`, or use `MIT` and `GPL-2.0` as license(s) first?